scripts/update-global-health-mpox.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+start_time=$(date +%s)
+
+echo '--- Update Mpox - Global Health'
+cd /home/owid/etl
```

Comment:
Could you update file `scripts/update-monkeypox.sh` instead and add those lines there? (i.e. it would be executing two snapshot files and doing single commit) It'll be nicer to have it in one place.

Reply:
Sure thing - I just added "poetry run python snapshots/health/latest/global_health_mpox.py" to the existing bash script and then deleted this one.